tox.ini

```diff
@@ -16,7 +16,7 @@ extras = tests
 commands = coverage run --parallel -m pytest {posargs}
 
 
-[testenv:py36]
+[testenv:py37]
```

Comment:
is that replacement intentional? the comment got out of sync

Reply:
do they? it says 3.6_+_ on purpose

Reply:
i can agree to that interpretation